### PR TITLE
fix(demos/kubeflow/e2e-kfp-mlflow-seldon-pipeline.ipynb): Fix seldon …

### PR DESCRIPTION
**Pull request by jneo8**
fix(demos/kubeflow/e2e-kfp-mlflow-seldon-pipeline.ipynb): Fix seldon deploy error with wrong conda_env

https://github.com/SeldonIO/seldon-core/issues/4378

### CODE REVIEW

File: demos/kubeflow/README.md [hunk 1]
```diff
@@ -35,6 +45,10 @@ $ juju config dex-auth static-password=admin
 
 The kubeflow dashboard will available at `$IPADDR.nip.io`
 
+```sh
```
Comment (chanchiwai-ray) [inline demos/kubeflow/README.md:48]: just a comment: did you used non-graphical vm that does not have access to browser?

File: demos/kubeflow/README.md [hunk 2]
```diff
@@ -72,7 +86,9 @@ If the values are not defined, apply a new user/password with the juju commands.
 
 ```sh
 $ juju config minio access-key=minio
-$ juju config minio secret-key=minio123
+$ kubectl get secret -n kubeflow minio-secret -o yaml | grep MINIO_SECRET_KEY | awk '{ print $2}' | base64 -d | xargs -I {} juju config minio secret-key={}
```
Comment (chanchiwai-ray) [inline demos/kubeflow/README.md:89]: Just a comment: can you make sure that MINIO_SECRET_KEY is available on the VM itself? I know It is available in the jupyter notebook with some check boxes checked, but not sure it's available on VM itself

**Comment by chanchiwai-ray** [comment]
Overall LGTM, when you check the comments and find no problem, you can merge it.

File: demos/kubeflow/README.md (hunk 1 above)
Comment (jneo8) [reply demos/kubeflow/README.md:48]: I think it depends on user's environment & network settings. I put command here just for those people who can't access the GUI. It's not for everyone but it still useful if someone don't know how to sshuttle multipass vm.

File: demos/kubeflow/README.md (hunk 2 above)
Comment (jneo8) [reply demos/kubeflow/README.md:89]: Not very sure about the question here. Which VM you mention here?

Comment (chanchiwai-ray) [reply demos/kubeflow/README.md:89]: nvm, I should read the whole command.

**Pull request merged**